src/wp-admin/includes/file.php

```diff
@@ -1563,6 +1563,37 @@ function wp_trusted_keys() {
 	return apply_filters( 'wp_trusted_keys', $trusted_keys );
 }
 
+/**
+ * Determines whether the given file is a valid ZIP file.
+ *
+ * This function does not test to ensure that a file exists. Non-existent files
+ * are not valid zips, so those will also return false.
```

Comment:
```suggestion
 * are not valid ZIPs, so those will also return false.
```

So colin doesn't give me a third strike.

Reply:
You're lucky!